package.json

```diff
@@ -2,26 +2,19 @@
   "name": "dota2",
   "version": "0.7.1",
   "dependencies": {
-    "protobuf": "*",
-    "deferred": "*",
-    "bignumber.js": "2.0.5",
-    "merge": ">= 0.0.2"
-  },
-  "peerDependencies": {
-    "steam": ">= 0.6.6"
+    "bignumber.js": "2.0.7",
+    "deferred": "^0.7.2",
+    "merge": "^1.2.0",
+    "protobufjs": "^4.0.0",
+    "steam": "^1.0.0"
   },
   "readmeFilename": "README.md",
   "description": "A node-steam plugin for Dota 2.",
   "main": "index.js",
-  "repository": {
```

Comment:
I really haven't. ;( 

No time to play as of late. I will at least try to review PRs and action them. There is a bit of backlog to review, much of which is superfluous now. Might sit down and do a library review this weekend or refamiliarise myself with the codebase, if another metaphorical fire doesn't crop up in the intervening period of time.

Also... Stop grafittiing these code comments, and kindly open an issue Kappa :fist: